java-checks/src/test/files/checks/IfElseIfStatementEndsWithElseCheck.java

```diff
@@ -0,0 +1,42 @@
+class A{
+  private static final boolean FALSE = false;
+  private int x = 2;
+
+  void foo() {
+    if (x == 0) {       // Compliant
+    } else if (x == 1) {
+        if(x == 2) {  // Noncompliant
+
+        }
+        else if(x == 3) {
```

Comment:
can you please reformat this test file to have something a bit more compact and coherent in term of style? You can also remove empty lines.